app/Http/Controllers/ProductController.php

```diff
@@ -84,19 +86,21 @@ public function postEdit($id, Request $request)
         $productModel = new Product;
         $this->validate($request,
             [
-                'txtName' => 'required|max:100',
-                Rule::unique('products')->ignore($productModel->id),
+                'txtName' => 'required|max:100|regex:/(^[A-Za-z0-9 ]+$)+/',
+                'txtName' => 'unique:products,name,' . $id,
                 'txtPrice' => 'required|numeric|max:10000000000',
-                'txtDescription' => 'max:300',
+                'txtDescription' => 'max:300|regex:/(^[A-Za-z ]+$)+/',
```

Comment:
You can only have letters in the description? What about numbers, ".,-?!/", etc..?